### ISSUE TITLE
how to change video play speed ?

### ISSUE THREAD

**Issue by yokins** [bradmartin/nativescript-videoplayer#131]
how to change video play speed ?

i want change the speed of the video. like 1.5x\2x\5x three speed.

**Comment by bradmartin** [comment]
I don't think this is currently supported. I do recall adding it to audio plugin, seen [here](https://github.com/nstudio/nativescript-audio/blob/master/src/android/player.ts#L258). 

A PR implementing this on android/ios would be welcomed and I can help with any questions you have.

**Comment by yokins** [comment]
👍 @bradmartin

**Comment by szhatef** [comment]
Is there update on having change play speed for videos on {N}?

**Comment by zangarmarsh** [comment]
For anyone who's struggling with player speed, for Android just add into videoplayer.android.js the following lines:
`
    Video.prototype.changeSpeed = function (speed) {
        this.player.setPlaybackParams( this.player.getPlaybackParams().setSpeed(speed) );
    }
`

For iOS the code - according to the specific player documentation - should be as above. I can't test it though, if any apple user can confirm I'll make a PR soon.

` Video.prototype.changeSpeed = function (speed) {
        this.player.setPlaybackParams( this.player.getPlaybackParams().setSpeed(speed) );
    }`
